### ISSUE TITLE
Bugs on the model upload front end

### ISSUE THREAD

**Issue by cristiangns** [dafnifacility/user-feedback#51]
I have recently uploaded the first version of my model and I encountered some bugs while I was uploading the docker image and the model definition file.

1. If there is a formatting issue with the model definition file you get the following error message: 
> Something went wrong while processing the model upload. Please wait a few minutes and try again

 This should be changed to more specific error messages that point to the actual error.

2. If you select the .tar and .yml files and then correct errors in the .yml file - removing the old .yml and uploading a new one gives the old error messages + new ones (if there are any). Sometimes you might get empty error messages that will still prevent you from uploading the model. You need to refresh the page and load both the .tar and .yml files again to be able to upload the model.

3. If you are using invalid characters for the model name - there error message is vague and it doesn't tell you what the problem is.

4. Since a Login session only lasts 1 hour - larger models might not finish uploading in time. If that happens then the upload process goes to 100% and then it remains there indefinitely. There is no error message but the model is not uploaded either.

**Comment by tg137** [comment]
Thanks a lot for the detailed feedback here Cristian.

I have created separate internal tasks to fix this stuff which are as follows:
1 + 3 - [CORE-1546](https://jira.dafni.rl.ac.uk/browse/CORE-1546) -> Issues in model definition YAML result in a 500 response and non-descriptive error message on the front end
2. [CORE-1547](https://jira.dafni.rl.ac.uk/browse/CORE-1547) -> Uploading an invalid definition file, fixing and reuploading still results in error messages
4. [CORE-1548](https://jira.dafni.rl.ac.uk/browse/CORE-1548) -> Update JWT expiry time to be 4 hours

**Comment by jiphex** [comment]
We've encountered [CORE-1546](https://jira.dafni.rl.ac.uk/browse/CORE-1546) when another user hit a YAML syntax issue (below) - for now I've advised the user to use e.g https://yamlchecker.com/ to check YAML syntax before uploading


```python
yaml.scanner.ScannerError: while scanning for the next token
found character '\t' that cannot start any token
  in "<file>", line 61, column 3
```

**Comment by roseDickinson** [comment]
These 3 bugs have been fixed and were included in the last release (v1.14.0)
